task2/js/script.js

```diff
@@ -0,0 +1,130 @@
+function isArray(something) {
+  return something instanceof Array;
+}
+
+function isArray2(something) {
+  return Object.prototype.toString.call(something) === '[object Array]';
+}
+
+function range(from, to, step) {
```

Comment:
range(10) -> [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10] 10 is wrong
range(-20, -10) -> [-20, -19, -18, -17, -16, -15, -14, -13, -12, -11, -10] -10 is wrong
range(10, 1, -3) -> [10, 7, 4, 1] 1 is wrong
basically looks good, you can document it's like [a, b] instead of [a;b)